NEWS.md

```diff
@@ -83,6 +83,7 @@ Standard library changes
 * `intersect` on `CartesianIndices` now returns `CartesianIndices` instead of `Vector{<:CartesianIndex}` ([#36643]).
 * `push!(c::Channel, v)` now returns channel `c`. Previously, it returned the pushed value `v` ([#34202]).
 * `RegexMatch` objects can now be probed for whether a named capture group exists within it through `haskey()` ([#36717]).
+* For consistency `RegexMatch` objects can also probed by `haskey` with an integer index for if a capture group exists within it ([#37300]).
```

Comment:
```suggestion
* `haskey(r::RegexMatch, i::Int) has been added and returns if the capture group for `i` exists ([#37300]).
```

maybe? "probed by `haskey`" got me a bit confused. Sometimes code can be easier to understand than words.

Reply:
```suggestion
* For consistency `haskey(r::RegexMatch, i::Integer) has also been added and returns if the capture group for `i` exists ([#37300]).
```